app/controllers/team_members_controller.rb

```diff
@@ -1,6 +1,8 @@
 class TeamMembersController < ApplicationController
 
   def index
-    @team_members = TeamMember.order(name: :asc)
+    @team_members = TeamMember.where(is_leader: false).order(name: :asc)
+    @leaders = TeamMember.where(is_leader: true).order(name: :asc)
+    @individual = TeamMember.where(id: 1)
```

Comment:
This is a code smell because of the hardcoded ID. Also, `@individual` is not used anywhere.